src/main/java/klerer/citibike/FindStations.java

```diff
@@ -0,0 +1,71 @@
+package klerer.citibike;
+
+import klerer.citibike.json.Station;
+
+import java.util.Objects;
+
+public class FindStations {
+
+    public Station stationById(Station[] stations, String stationId) {
+        Station foundStation = null;
+        for (Station station : stations) {
+            if (Objects.equals(stationId, station.station_id)) {
+                foundStation = station;
+                return station;
```

Comment:
Do you need foundStation anymore?

Reply:
no, but what do I return at the end?

Reply:
What do you return right now if there isn't a station to return?

Reply:
now they are just returning null and my tests are failing

Reply:
Run it through the debugger and figure out why they are failing

Reply:
I cant because of that error i am getting that i emailed you about